android/app/src/main/java/com/microsoft/codepush/common/core/CodePushListeners.java

```diff
@@ -0,0 +1,37 @@
+package com.microsoft.codepush.common.core;
+
+import com.microsoft.codepush.common.interfaces.CodePushBinaryVersionMismatchListener;
+import com.microsoft.codepush.common.interfaces.CodePushDownloadProgressListener;
+import com.microsoft.codepush.common.interfaces.CodePushSyncStatusListener;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Incapsulates listeners that {@link CodePushBaseCore} is using.
+ */
+public class CodePushListeners {
+    /**
+     * List of {@link CodePushSyncStatusListener}.
+     */
+    public final List<CodePushSyncStatusListener> mSyncStatusListeners;
```

Comment:
Is there a reasons it is a `List`, not just one listener? Is it used somewhere?

Reply:
@annakocheshkova Actually, we could have more than one listeners on native side, depends on user wish.